libs/ngx-gauge/src/gauge/gauge.ts

```diff
@@ -257,7 +259,7 @@ export class NgxGauge implements AfterViewInit, OnChanges, OnDestroy {
         }
         function animate(timestamp) {
             timestamp = timestamp || new Date().getTime();
-            let runtime = timestamp - startTime;
+            let runtime = self.animate ? timestamp - startTime : duration;
```

Comment:
In order to add animate option I would like to remove calling requestAnimationFrame altogether. It means that avoid calling animate function when animate property is is switched off.